Cargo.toml

```
@@ -17,8 +17,8 @@ backtrace = "0.3.0"
 rand = "0.8.0"
 hex = "0.4.3"
 tracing = "0.1"
-libcrux-kem = {version = "0.0.2-alpha.1", features = ["pre-verification", "kyber"]}
-libcrux = { version = "0.0.2-alpha.1", features = [
+libcrux-kem = { git = "https://github.com/cryspen/libcrux", features = ["pre-verification", "kyber"]}
```

No pre-verification needed anymore.

```suggestion
libcrux-kem = { git = "https://github.com/cryspen/libcrux", features = ["kyber"]}
```